### PR TITLE
Alert

### PR DESCRIPTION
**Pull request by vermasavita**
I have created nav bar & alert comp. 

Kindly review this.

### CODE REVIEW

File: css/components.css [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+}
+
+.sub-head-content{
+    font-size:1.7rem;
+    line-height: 25px;
+    margin-top: 1rem;
+}
+
+.practical{
+    box-shadow: rgba(0, 0, 0, 0.02) 0px 1px 3px 0px, rgba(27, 31, 35, 0.15) 0px 0px 0px 1px;
```
Comment (mohammed-sas) [inline css/components.css:42]: why are you defining box-shadow props two times

File: css/nav.css [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+}
+
+.toggle-btn{
+    position: absolute;
+    top: 1rem;
+    right: 1rem;
+    display: none;
+    flex-direction: column;
+    justify-content: space-between;
+    width: 35px;
```
Comment (mohammed-sas) [inline css/nav.css]: it would be better if you follow relative units convention throughout

**Review by mohammed-sas — CHANGES_REQUESTED** [review]
following relative units throughout is a good practise

File: index.html [hunk 3]
```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>UI Book</title>
+    <link rel="stylesheet" href="nav.css">
```
Comment (kart-c) [inline index.html]: Hi Savita can you check this `href`? I think the path is wrong

File: css/nav.css (hunk 2 above)
Comment (vermasavita) [reply css/nav.css]: Yea, agree. Changed it. thanks :-)

File: css/components.css (hunk 1 above)
Comment (vermasavita) [reply css/components.css:42]: I have used multiple shadows here.

**Review by kart-c — CHANGES_REQUESTED** [review]

File: index.html (hunk 3 above)
Comment (vermasavita) [reply index.html]: Oh, yes. corrected it. Thanks :-)

Any other feedback?

File: components/avatar/avatar.html [hunk 4]
```diff
@@ -0,0 +1,127 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Alert</title>
```
Comment (kart-c) [inline components/avatar/avatar.html:7]: Change the title for avatar element

File: components/avatar/avatar.html [hunk 5]
```diff
@@ -0,0 +1,127 @@
...
+                <div class="alert info">
+                  
+                  <i class='bx bx-info-circle'></i>
+                  This is an info alert — check it out!
+                
+                </div>
+                <div class="alert success">
+                  
+                  <i class='bx bx-check-circle'></i>
+                  This is a success alert — check it out!
```
Comment (kart-c) [inline components/avatar/avatar.html:95]: If this is unnecessary you should remove it

**Review by kart-c — APPROVED** [review]
I have suggested 2 changes. When done you can go ahead and merge the branch

**Review by shaik-saad — APPROVED** [review]
Good work. Although I see some commented code, if it's not relevant then remove it.
One more suggestion, you're using two fonts and also importing them in two lines which can be done in one line, so try to import them in single line.

File: components/avatar/avatar.html (hunk 4 above)
Comment (vermasavita) [reply components/avatar/avatar.html:7]: Done

File: components/avatar/avatar.html (hunk 5 above)
Comment (vermasavita) [reply components/avatar/avatar.html:95]: Removed

**Pull request merged**